### DISCUSSION TITLE
Are there any flash files for the rating sprites/grafitti sprites

### DISCUSSION THREAD

**Discussion by libauror-a** [FunkinCrew/Funkin#1841] [Q&A] [answered]
Are there any flash files for the rating sprites/grafitti sprites, or are they made using only PNG's?

**Comment by ninjamuffin99** [comment, marked as answer]
they were made by evilsk8r in the art program he uses, which is NOT flash. So now flash files for em!

**Reply by libauror-a** [reply]
thanks muffin guy
